src/renderer/pages/game-details/hero-panel.tsx

```diff
@@ -27,6 +28,8 @@ export function HeroPanel({
 }: HeroPanelProps) {
   const { t } = useTranslation("game_details");
 
+  const [showBinaryNotFoundModal, setShowBinaryNotFoundModal] = useState<boolean>(false);
```

Comment:
No need tao pass `boolean` as a generic, since TypeScript can infer the type from the default value